aspnetcore/security/authentication/windowsauth.md

```diff
@@ -57,6 +57,8 @@ dotnet new webapp --auth Windows
 
 ---
 
+When manually modifying an existing project, confirm that the app's project file includes a package reference for the [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app) **or** the [Microsoft.AspNetCore.Authentication](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication/) NuGet package.
```

Comment:
Confirm the project file includes a package reference for [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app) **or** [Microsoft.AspNetCore.Authentication](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication/).